nengo_gui/components/__init__.py

```diff
@@ -1,9 +1,21 @@
-from .slider import Slider, SliderTemplate
-from .value import Value, ValueTemplate
-from .xyvalue import XYValue, XYValueTemplate
-from .sim_control import SimControl, SimControlTemplate
-from .raster import Raster, RasterTemplate
-from .voltage import Voltage, VoltageTemplate
-from .pointer import Pointer, PointerTemplate
-from .netgraph import NetGraph, NetGraphTemplate
-from .ace_editor import AceEditor, AceEditorTemplate
+import inspect
+import sys
+
+from .component import Component
+from .slider import Slider
+from .value import Value
+from .xyvalue import XYValue
+from .sim_control import SimControl
+from .raster import Raster
+from .voltage import Voltage
+from .pointer import Pointer
+from .netgraph import NetGraph
+from .ace_editor import AceEditor
+
+# Old versions of the .cfg files used Templates which had slightly different
+# names than the Components currently usef.  This code is needed to
+# successfully parse those old .cfg files
+this_module = sys.modules[__name__]
```

Comment:
That is a really good point.  :)  I've updated it to use your much clearer approach.